### PR TITLE
addButton

### PR DESCRIPTION
**Pull request by mergeflydemo**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const [count, setCount] = useState(0);
```
Comment (avi-mergefly) [inline src/App.js]: Nice Job!

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
```
Comment (avi-mergefly) [inline src/App.js]: Nice Job!

Comment (avi-mergefly) [inline src/App.js]: Nice

File: src/App.js [hunk 3]
```diff
@@ -19,6 +21,9 @@ function App() {
           Learn React
         </a>
       </header>
+      <button onClick={()=>setCount(count + 1)}>Increment</button>
+      I've been clicked {count} times!
```
Comment (avi-mergefly) [inline src/App.js]: Looks Great!!

File: src/App.js [hunk 4]
```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
```
Comment (avi-mergefly) [inline src/App.js]: Looks Great!

File: src/App.js [hunk 5]
```diff
@@ -19,6 +21,9 @@ function App() {
           Learn React
         </a>
       </header>
+      <button onClick={()=>setCount(count + 1)}>Increment</button>
```
Comment (eprince1) [inline src/App.js]: test

Comment (avi-mergefly) [inline src/App.js]: Can you add some tests?

File: src/App.js (hunk 3 above)
Comment (avi-mergefly) [inline src/App.js]: Can you add tests?

Comment (avi-mergefly) [inline src/App.js]: Looks Great!

Comment (mergeflydemo) [reply src/App.js]: Thanks!!

File: src/App.js [hunk 6]
```diff
@@ -1,8 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const [count, setCount] = useState(0);
+
   return (
```
Comment (eprince1) [inline src/App.js]: comment

File: src/App.js [hunk 7]
```diff
@@ -1,8 +1,10 @@
...
     <div className="App">
```
Comment (avisprince) [inline src/App.js]: test

File: src/App.js [hunk 8]
```diff
@@ -1,8 +1,10 @@
...
       <header className="App-header">
```
Comment (avisprince) [inline src/App.js]: test

File: src/App.js (hunk 7 above)
Comment (avisprince) [inline src/App.js]: test

**Pull request closed without merging**